src/Core/Form/IdentifiableObject/CommandBuilder/Product/Combination/DefaultCombinationCommandsBuilder.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\Combination;
+
+use PrestaShop\PrestaShop\Core\Domain\Product\Combination\Command\SetDefaultCombinationCommand;
+use PrestaShop\PrestaShop\Core\Domain\Product\Combination\ValueObject\CombinationId;
+
+class DefaultCombinationCommandsBuilder implements CombinationCommandsBuilderInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function buildCommands(CombinationId $combinationId, array $formData): array
+    {
+        if (!isset($formData['is_default'])) {
```

Comment:
```suggestion
        if (!isset($formData['is_default']) || $formData['is_default'] === false) {
```

Reply:
```suggestion
        if (empty($formData['is_default'])) {
```

Reply:
Shouldn't we test that the value is present AND is true? I know that in forms with checkboxes when it's unchecked the value is not present at all but since the builder may be used in another context it's probably safer to check the value as well

Because with the current code I believe that `$data = ['is_default' => false];` will return the command

Reply:
But this should be tested in a unit test anyway, which are missing for now